inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/roi_pooling.cpp

```diff
@@ -12,7 +12,7 @@ using namespace LayerTestsDefinitions;
 namespace {
 
 const std::vector<std::vector<size_t>> inShapes = {
-    {3, 4, 50, 50}
+    {1, 4, 50, 50}
```

Comment:
The first dimension does not necessarily have to be 1. Its value may be different. Please make sure plugin implementation works also for this case.

Reply:
Myriad plugin does not support `batch`!=1. That is the reason I have changed the batch.